src/plugin/config_test.go

```diff
@@ -72,18 +73,27 @@ func TestFailOnMissingRequiredEnvironment(t *testing.T) {
 func TestFetchConfigFromEnvironment(t *testing.T) {
 	unsetEnv(t, "BUILDKITE_PLUGIN_ECS_TASK_RUNNER_PARAMETER_NAME")
 	unsetEnv(t, "BUILDKITE_PLUGIN_ECS_TASK_RUNNER_SCRIPT")
+	unsetEnv(t, "BUILDKITE_PLUGIN_ECS_TASK_RUNNER_TIME_OUT")
 
 	var config plugin.Config
 	fetcher := plugin.EnvironmentConfigFetcher{}
 
 	t.Setenv("BUILDKITE_PLUGIN_ECS_TASK_RUNNER_PARAMETER_NAME", "test-parameter")
 	t.Setenv("BUILDKITE_PLUGIN_ECS_TASK_RUNNER_SCRIPT", "hello-world")
+	t.Setenv("BUILDKITE_PLUGIN_ECS_TASK_RUNNER_TIME_OUT", "600")
 
 	err := fetcher.Fetch(&config)
 
 	require.NoError(t, err, "fetch should not error")
 	assert.Equal(t, "test-parameter", config.ParameterName, "fetched message should match environment")
 	assert.Equal(t, "hello-world", config.Script, "fetched message should match environment")
+	assert.Equal(t, 600, config.TimeOut, "fetched message should match environment")
```

Comment:
whoopies, I made a mistake in one of these messages and it got you too.

```suggestion
	assert.Equal(t, "hello-world", config.Script, "fetched script should match environment")
	assert.Equal(t, 600, config.TimeOut, "fetched timeout should match environment")
```